tests/models_test.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for models."""
+from unittest.mock import patch
+from datetime import datetime
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.utils import timezone
+
+from blog.models import Post
+
+
+class TestPost(TestCase):
+    """Main class for testing models."""
+
+    def setUp(self):
+        """Prepare data for testing."""
+        self.user = User.objects.create(username='testuser')
+        self.test_post = (
+            Post.objects.create(
+                author=self.
```

Comment:
This is a weird place to interrupt a line.

Reply:
Wow, locally I don't have interrupt